aether-utils/aether-client/aether/client_test/fixtures/__init__.py

```diff
@@ -29,60 +27,56 @@ def file_to_json(path):
 
 #Projects
 project_name = "TestProject"
-project_def = file_to_json(here+"/fixtures/salad/salad.json")
-project_obj = {
+project_template = {
     "revision": "1",
-    "name": project_name,
-    "salad_schema": project_def,
+    "name": None,
+    "salad_schema": "{}",
```

Comment:
These fields are not longer mandatory, if you are not using them in the client, I think it's better to remove them here (simple it's better).
`salad_schema`, `jsonld_context`, `rdf_definition`